### ISSUE TITLE
code run stop early at 40 epochs

### ISSUE THREAD

**Issue by lee29321** [neurosim/MLP_NeuroSim_V3.0#14]
I am running the script with real device 1T1R with Ag:a-Si case. 
I am trying to run the training to the 125 epochs. After I compile the code and run, the output would stop at 35-40 epochs without sending any error message. The accuracy until the end of the epoch is expected. 

I have tried to reduce the batch size or change the learning rate. However, it doesn't solve the problem and the accuracy went low to 50%. I also tried to change the learning rate of alpha and beta, however, the total epochs  still maintains 38-40 epochs. 

Could you please let me know how to change the condition to make the code run for 125 epochs?
